_migration-assistant/migration-phases/removing-migration-infrastructure.md

```diff
@@ -7,15 +7,22 @@ parent: Migration phases
 
 # Removing migration infrastructure
 
-After a migration is complete all resources should be removed except for the target cluster, and optionally your Cloudwatch Logs, and Traffic Replayer logs.
+After a migration is complete, you should remove all resources except for the target cluster, and optionally your Amazon CloudWatch Logs and Traffic Replayer logs.
 
-To remove all the CDK stack(s) which get created during a deployment you can execute a command similar to below within the CDK directory
+To remove all the AWS Cloud Development Kit (AWS CDK) stack(s) created during a deployment, run the following command within the CDK directory:
 
-```bash
+```bash  
+cd deployment/cdk/opensearch-service-migration
 cdk destroy "*" --c contextId=<CONTEXT_ID>
 ```
 {% include copy.html %}
 
-Follow the instructions on the command-line to remove the deployed resources from the AWS account.
+Follow the instructions on the command line to remove the deployed resources from your AWS account.
 
-The AWS Management Console can also be used to remove Migration Assistant resources and confirm that they are no longer in the account.
\ No newline at end of file
+You can also use the AWS Management Console to remove Migration Assistant resources and confirm that they are no longer in the account.
+
+## Uninstalling Migration Assistant for Amazon OpenSearch Service
+
+You can uninstall the Migration Assistant for Amazon OpenSearch Service solution from the AWS Management Console or by using the AWS Command Line Interface (AWS CLI). Manually remove the contents of the Amazon S3 bucket that matches cdk-<unique id>-assets-<account id>-<region> created by this solution. Migration Assistant for Amazon OpenSearch Service does not automatically delete Amazon S3 buckets in case you have stored data to retain. 
+
```

Comment:
Line 26: Please rewrite the second sentence for clarity (the text surrounding cdk-<unique id>-assets-<account id>-<region> is confusing).